src/SqlSquared/Parser.purs

```diff
@@ -676,63 +543,33 @@ tableRelation = PC.try do
        (pure ∘ E.Left ∘ (Pt.rootDir Pt.</> _))
        ∘ Pt.sandbox Pt.rootDir)
       i
-
   a ← PC.optionMaybe do
     _ ← keyword "as"
     ident
   pure $ Sig.TableRelation { alias: a, path }
 
 
-variRelation
-  ∷ ∀ t m
-  . Corecursive t (Sig.SqlF EJ.EJsonF)
-  ⇒ Monad m
-  ⇒ P.ParserT (Array Token) m (Sig.Relation t)
-variRelation = PC.try do
+variRelation ∷ ∀ m t. SqlParser m t (Sig.Relation t)
+variRelation = do
   operator ":"
   vari ← ident
```

Comment:
Do you mean `create function` is an expression in the scala parser or in this one? It shouldn't be in this one at least because they are only allowed in `SqlQuery` and `SqlModule`. I don't know about tokenizing varis. There are actually two problems:

* Currently, `:    foo` is a valid vari, which I think probably shouldn't be possible.
* If we tokenize vari, what do we do about `{ foo:bar }`. That's a valid map but would be tokens `{, foo, :bar, }`.

I don't think the backend parser tokenizes at all. Is there a reason that we don't just parse directly from the source string? I don't think we are getting anything interesting from our tokenizer.